samples/graph-teams-tag/nodejs/README.md

```diff
@@ -0,0 +1,123 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates how to use CRUD Graph operations related to team tags.
+products:
```

Comment:
Pleaser verify if all the images are visible properly in Readme file.

Reply:
Yes they are visible